gymnasium/wrappers/monitoring/video_recorder.py

```diff
@@ -46,16 +50,12 @@ def __init__(
                 "MoviePy is not installed, run `pip install moviepy`"
             )
 
-        self._async = env.metadata.get("semantics.async")
-        self.enabled = enabled
-        self._closed = False
-
         self.render_history = []
         self.env = env
 
         self.render_mode = env.render_mode
 
-        if "rgb_array_list" != self.render_mode and "rgb_array" != self.render_mode:
+        if "rgb_array" != self.render_mode and "rgb_array_list" != self.render_mode:
```

Comment:
Since we're changing this, can you flip the inequalities? Doing `<literal> != <variable>` makes me uneasy for a reason that stems either from some other language, of from some ancient evil hidden in the depths of python magic methods.

Reply:
We could also be even more clever and do `if self.render_mode not in ("rgb_array", "rgb_array_list")`

Reply:
Changed